cirq/circuits/circuit.py

```diff
@@ -659,6 +660,50 @@ def findall_operations_between(self,
 
         return list(result)
 
+    def findall_operations_until_blocked(self,
+                                   start_frontier: Dict[ops.QubitId, int],
+                                   is_blocker: Callable[[ops.Operation], bool]
+                                         = lambda op: False,
+                                   omit_crossing_operations: bool = False
+                                   ) -> List[Tuple[int, ops.Operation]]:
+        """
+        Finds all operations until a blocking operation is hit.  For each
+        qubit, this returns a list of all operations that involve that qubit
+        from the starting moment until a blocking operation that involves that
+        qubit.  Operations not involving that qubit are not considered blocking
+        for that qubit, even if they affect other qubits listed in the
+        start_frontier dictionary.  See reachable_frontier_from for a more
```

Comment:
Use back ticks when referring to code elements such as `reachable_frontier_from`. They get used by the documentation system.